marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/numericFacetValuesRange/numericFacetValuesRange.sjs

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var entityName;
+var facetName;
+var indexType;
```

Comment:
Should be able to derive this from the entity model so that the client doesn't have to pass it in or care about it